### ISSUE TITLE
Disable recreateClosed for angular dependencies

### ISSUE THREAD

**Issue by spalger** [renovatebot/config-help#262]
### Which Renovate are you using? CLI, App, or Pro

App

### Which platform are you using? GitHub, GitLab, Bitbucket Azure DevOps

GitHub

### Have you checked the logs? Don't forget to include them if relevant

https://renovatebot.com/dashboard#github/elastic/kibana/70752142, relevant bits:
```
DEBUG: getBranchPr(renovate/angular)(dependencies=["@types/angular","angular","angular-aria","angular-mocks","angular-resource","angular-route","angular-sanitize"], branch="renovate/angular")
DEBUG: findPr(renovate/angular, null, open)(dependencies=["@types/angular","angular","angular-aria","angular-mocks","angular-resource","angular-route","angular-sanitize"], branch="renovate/angular")
DEBUG: branchExists=true(dependencies=["@types/angular","angular","angular-aria","angular-mocks","angular-resource","angular-route","angular-sanitize"], branch="renovate/angular")
DEBUG: Branch has 7 upgrade(s)(dependencies=["@types/angular","angular","angular-aria","angular-mocks","angular-resource","angular-route","angular-sanitize"], branch="renovate/angular")
DEBUG: recreateClosed is true(dependencies=["@types/angular","angular","angular-aria","angular-mocks","angular-resource","angular-route","angular-sanitize"], branch="renovate/angular")
DEBUG: Checking if PR has been edited(dependencies=["@types/angular","angular","angular-aria","angular-mocks","angular-resource","angular-route","angular-sanitize"], branch="renovate/angular")
DEBUG: Checking schedule(at any time, null)(dependencies=["@types/angular","angular","angular-aria","angular-mocks","angular-resource","angular-route","angular-sanitize"], branch="renovate/angular")
```

### What is your question?

I have tried specifying `recreateClosed=false` both [globally](https://github.com/elastic/kibana/blob/6cecba85ea516fae7997b1c4aea1f42915994b6f/renovate.json5#L197-L201) and in the angular [packageGroup](https://github.com/elastic/kibana/blob/610c132bcc50db23c7b1d1126efb32e53af16500/renovate.json5#L109-L114) but renovatebot still insists that "recreateClosed is true", any ideas?

**Comment by rarkins** [comment]
I may be able to work around this. Do you have an example PR?

**Comment by spalger** [comment]
https://github.com/elastic/kibana/pull/37884

**Comment by rarkins** [comment]
If the types update wasn’t included then it would be classified as all v1.7.8 and would be closeable. Right now it’s “immortal” because Renovate can’t determine a common version.

**Comment by spalger** [comment]
Interesting, okay, we can just keep the PR open for now. Appreciate you looking into it!
